src/corelibs/U2Designer/src/wizard/WizardController.cpp

```diff
@@ -66,11 +66,11 @@
 
 namespace U2 {
 
-WizardController::WizardController(Schema *s, const Wizard *w)
-    : QObject(), schema(s), wizard(w), runAfterApply(false) {
+WizardController::WizardController(std::shared_ptr<Schema> s, const Wizard *w)
```

Comment:
This code does not compile
I would suggest:
1) Use QT variants of the pointers. They have no cross platform/compiler issues.
2) When use std always use std prefix (see move())
3) For any non-primitive type method parameters (even shared structures) use (const Type&).